subprojects/docs/src/docs/userguide/jvm/java_testing.adoc

```diff
@@ -222,10 +222,16 @@ There is also a standalone link:{groovyDslPath}/org.gradle.api.tasks.testing.Tes
 
 .Creating a unit test report for subprojects
 ====
-include::sample[dir="snippets/testing/testReport/groovy",files="build.gradle[tags=test-report]"]
-include::sample[dir="snippets/testing/testReport/kotlin",files="build.gradle.kts[tags=test-report]"]
+include::sample[dir="snippets/testing/testReport/groovy",files="buildSrc/src/main/groovy/myproject.java-conventions.gradle[tags=test-report];build.gradle[tags=test-report]"]
+include::sample[dir="snippets/testing/testReport/kotlin",files="buildSrc/src/main/kotlin/myproject.java-conventions.gradle.kts[tags=test-report];build.gradle.kts[tags=test-report]"]
 ====
 
+In this example, we use a convention plugin `myproject.java-conventions` to expose the test results from a project to Gradle's <<variant_model.adoc#,variant aware dependency management engine>>.
+
+The plugin declares a consumable `binaryTestResultsElements` configuration that represents the binary test results of the `test` task.
+In the aggregation project's build file, we declare `testReportData` configuration and depend on all of the projects that we want to aggregate the results from. Gradle will automatically select the binary test result variant from each of the subprojects instead of the project's jar file.
```

Comment:
```suggestion
In the aggregation project's build file, we declare the `testReportData` configuration and depend on all of the projects that we want to aggregate the results from. Gradle will automatically select the binary test result variant from each of the subprojects instead of the project's jar file.
```